lib/_http_outgoing.js

```diff
@@ -165,9 +165,7 @@ ObjectDefineProperty(OutgoingMessage.prototype, '_headers', {
       this[kOutHeaders] = null;
     } else if (typeof val === 'object') {
       const headers = this[kOutHeaders] = ObjectCreate(null);
-      const keys = ObjectKeys(val);
-      for (var i = 0; i < keys.length; ++i) {
-        const name = keys[i];
+      for (const name of ObjectKeys(val)) {
```

Comment:
We should not make this change for performance reasons.

Reply:
resolved in https://github.com/nodejs/node/pull/30958/commits/17049d56621869d8776a922d5e56b6c5ec35be7b